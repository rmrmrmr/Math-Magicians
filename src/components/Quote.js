import React from 'react';
import { Link } from 'react-router-dom';

export default function Quote() {
  return (
    <div className="flex w-screen h-screen flex-wrap p-8">
      <h1 className="text-8xl w-4/5">Math Magicians</h1>
      <nav className="flex w-1/5 justify-between text-lg">
        <Link to="/">Home</Link>
        <p>|</p>
        <Link to="/calculator">Calculator</Link>
        <p>|</p>
        <Link to="/quote">Quote</Link>
      </nav>
      <div className="flex flex-col gap-y-6">
        <h2 className="text-4xl w-full text-center pt-30 px-9 pb-0 h-min">
          “What wonderful power there is in the real numbers,
          since one is in a position to determine uniquely,
          with a single coordinate, the elements of an n-dimensional continuous space.”
        </h2>
        <h2 className="text-3xl w-full text-center pt-0">
          -Georg Cantor
        </h2>
      </div>
    </div>
  );
}
