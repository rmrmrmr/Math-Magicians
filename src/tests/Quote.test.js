import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../components/Quote';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Quote />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
