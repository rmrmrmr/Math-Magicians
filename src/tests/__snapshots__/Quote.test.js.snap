// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex w-screen h-screen flex-wrap p-8"
>
  <h1
    className="text-8xl w-4/5"
  >
    Math Magicians
  </h1>
  <nav
    className="flex w-1/5 justify-between text-lg"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <p>
      |
    </p>
    <a
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <p>
      |
    </p>
    <a
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
  <div
    className="flex flex-col gap-y-6"
  >
    <h2
      className="text-4xl w-full text-center pt-30 px-9 pb-0 h-min"
    >
      “What wonderful power there is in the real numbers, since one is in a position to determine uniquely, with a single coordinate, the elements of an n-dimensional continuous space.”
    </h2>
    <h2
      className="text-3xl w-full text-center pt-0"
    >
      -Georg Cantor
    </h2>
  </div>
</div>
`;